vmdb/app/views/layouts/gtl/_list.html.haml

```diff
@@ -0,0 +1,353 @@
+- # New code to use DHTMLXGRID below
+- if @grid_xml
+  - if @showlinks == false
```

Comment:
can we just `if @showlinks` and swap the branches?

also the `url = ` could be before the condition